src/schema/resolvers/user/mutation.resolver.js

```diff
@@ -0,0 +1,39 @@
+import { ApolloError } from "apollo-server-errors";
+import { hash } from "utils/auth";
+
+const updateUser = async (__, args, context) => {
+  const password = await hash(args.password);
```

Comment:
Since password is an optional argument, it may be undefined or null so might be good to check if it has value first before hashing